### DISCUSSION TITLE
Log retention confusion

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#5556] [Unsupported Versions]
Hello!

I want to limit the Elasticsesarch log retention to 200 days and I'm confused about the documentation vs. the default settings on my manager node. cur_close_days is set to null in my manager minion config file, instead there are plenty of index_settings in the global.sls like

index_settings:
      so-beats:
        shards: 1
        warm: 7
        close: 30
        delete: 365
...
I was not able to find a documentation of these settings, but my understanding ist, that ES logs get deleted after 365 days. Correct?

Additionally log_size_limit is set to 474. Is this MB, GB, TB, ...? Thanks in advance!

Cheers, Ben

**Comment by dougburks** [comment]
log_size_limit is measured in GB:
https://github.com/Security-Onion-Solutions/securityonion/blob/39f32a6e138ed29a4515057fec00524d8887c398/setup/so-whiptail#L596

So the first thing you'll need to do is make sure that you have enough disk space for 200 days of log retention and that log_size_limit is set appropriately.

**Reply by ben-sec** [reply]
Disk space: 10.5 TB. Check
log_size_limit: 8582 GB Check
So it only works by fine tuning the log_size_limit to meet the maximum retention time (assuming that I have mounted enough disk space)? But what is the meaning of the mentioned index_settings in global.sls?

Thanks Doug!

Cheers, Ben

**Reply by TOoSmOotH** [reply]
The delete option will only be honored starting in .80 although the reason it is set to 365 is so that the disk space cleanup takes care of it. This setting will be more important if you were running a true cluster.
